homeassistant/components/zwave_js/switch.py

```diff
@@ -152,3 +155,30 @@ def _update_state(self) -> None:
             self._state = (
                 self.info.primary_value.value == BarrierEventSignalingSubsystemState.ON
             )
+
+
+class ZWaveConfigParameterSwitch(ZWaveSwitch):
+    """Representation of a Z-Wave config parameter switch."""
+
+    _attr_entity_category = EntityCategory.CONFIG
+
+    def __init__(
+        self, config_entry: ConfigEntry, driver: Driver, info: ZwaveDiscoveryInfo
+    ) -> None:
+        """Initialize a ZWaveConfigParameterSwitch entity."""
+        super().__init__(config_entry, driver, info)
+
+        property_key_name = self.info.primary_value.property_key_name
+        # Entity class attributes
+        self._attr_name = self.generate_name(
+            alternate_value_name=self.info.primary_value.property_name,
+            additional_info=[property_key_name] if property_key_name else None,
+        )
+
+    async def async_turn_on(self, **kwargs: Any) -> None:
+        """Turn the switch on."""
+        await self.info.node.async_set_value(self.info.primary_value, 1)
```

Comment:
Can this raise any exceptions?

Reply:
theoretically set value can raise an exception regardless of where we use it. If we want to add handling for that, we probably need to do it in other platforms as well

Reply:
I think we can start here now since it's new code. We want to raise HomeAssistantError on api errors.